test/unittests/commands/create/create_on_server/test_project.py

```diff
@@ -0,0 +1,172 @@
+# mypy: disable-error-code="no-untyped-def"
+
+from http import HTTPStatus
+from unittest.mock import Mock
+from unittest.mock import patch
+
+import pytest
+
+from dsp_tools.commands.create.create_on_server.project import create_project
+from dsp_tools.commands.create.models.parsed_project import ParsedProjectMetadata
+from dsp_tools.error.exceptions import ProjectNotFoundError
+from dsp_tools.error.exceptions import UnableToCreateProjectError
+from dsp_tools.utils.request_utils import ResponseCodeAndText
+from test.unittests.commands.create.constants import PROJECT_IRI
+
+
+@pytest.fixture
+def mock_auth() -> Mock:
+    auth = Mock()
+    auth.server = "http://0.0.0.0:3333"
+    return auth
+
+
+@pytest.fixture
+def parsed_project() -> Mock:
+    project = Mock(spec=ParsedProjectMetadata)
+    project.shortcode = "9999"
+    return project
+
+
+@pytest.fixture
+def existing_project_iri() -> str:
+    return PROJECT_IRI
+
+
+@pytest.fixture
+def new_project_iri() -> str:
+    return "http://rdfh.ch/projects/newProjectIRI"
```

Comment:
Here my personal preference would be to use a global constant (1 LOC) instead of a fixture (3 LOC). But it doesn't matter that much, you can also leave it.